package.json

```diff
@@ -5,7 +5,6 @@
   "private": true,
   "type": "commonjs",
   "scripts": {
-    "prepare": "is-ci || husky install",
```

Comment:
Can we use `postinstall` instead of hacking `yarn.config.js` for this use case? We don't have the same problem as Prettier, because our users don't use our top-level package.json.

Reply:
Good idea!